api/iode.h

```diff
@@ -50,9 +50,9 @@
 //#include "o_objs.h" // JMP 8/12/2011
 
 /******************************* DEFINES **********************************/
-#define IODE_VERSION "IODE Modeling Software 7.0 - (c) 1990-2023 Federal Planning Bureau - Brussels"
-#define IODE_VERSION_MAJOR 7
-#define IODE_VERSION_MINOR 0
+#define IODE_VERSION "IODE Modeling Software 6.70 - (c) 1990-2023 Federal Planning Bureau - Brussels"
```

Comment:
Do we change to 6.70 in the public iode repo or do we stay with 7.0 ? 
I hesitate. 
What do you think ?

For the moment, the Qt GUI says it this the 7.0 version to the users when they start it.